Sources/KeychainSwift.swift

```diff
@@ -95,13 +97,28 @@ open class KeychainSwift {
     let accessible = access?.value ?? KeychainSwiftAccessOptions.defaultOption.value
       
     let prefixedKey = keyWithPrefix(key)
+    var accessWithFlag:Any = accessible as Any
```

Comment:
```suggestion
    let accessWithFlag: Any = accessible as Any
```

I think you can do 
```
let accessWithFlag: Any
```
and then
```
if (..) {
accessWithFlag = SecAccessControlCreateWithFlags(nil, accessible as CFString, control.value, nil) as Any
} else {
accessWithFlag = accessible as Any
}
```

Reply:
Fixed in: 067bb33c34c39595b6b117e014665a5c661bfca4